src/DynamoCoreWpf/Views/CodeBlocks/CodeBlockEditor.cs

```diff
@@ -7,12 +7,56 @@
 namespace Dynamo.UI.Controls
 {
     /// <summary>
+    /// Code block node editor state machine.
+    /// </summary>
+    internal class EditorStateMachine
+    {
+        internal enum EditorState
```

Comment:
The fully qualified name of this `EditorStateMachine.EditorState` can be made more succinct as `EditorStateMachine.State`, don't you think? But that's just my opinion.